route.go

```diff
@@ -396,7 +396,7 @@ func (r *Route) Schemes(schemes ...string) *Route {
 	for k, v := range schemes {
 		schemes[k] = strings.ToLower(v)
 	}
-	if r.buildScheme == "" && len(schemes) > 0 {
+	if r.getBuildScheme() == "" && len(schemes) > 0 {
```

Comment:
Exactly. The routing will only make it to the child route if the parents also include its schemes, but the build scheme of the parent is not necessarily the one that the child will want to build with, so its schemes should take precedence if they are defined.